### DISCUSSION TITLE
Exclude mount points to other filesystems like rsync's --one-file-system

### DISCUSSION THREAD

**Discussion by StanislavNikolov** [borgbase/vorta#822] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently it looks like Vorta follows mounts to other file systems, which is undesirable in my case, especially for sshfs through FUSE.

**Describe the solution you'd like**
I would like to have an option to not cross filesystem boundary, not just by listing mount points in exclude patterns. Rsync's `--one-file-system` does the same thing.

**Describe alternatives you've considered**
It is also possible to make a pre-backup command to unmount everything inside the to-be-backed-up path, but that will mess up my workflow if I request a backup while I work.

**Comment by m3nu** [comment]
Already possible with #677, but not ready to merge yet.
